src/sfctl/custom_deployment.py

```diff
@@ -0,0 +1,131 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -----------------------------------------------------------------------------
+
+"""Custom commands for managing Service Fabric Mesh resources"""
+
+from __future__ import print_function
+
+import enum
+import json
+import os
+import shutil
+from knack.util import CLIError
+from sfmergeutility import SFMergeUtility
+
+class ResourceType(enum.Enum):
+    """ Defines the valid yaml resource types
+        which are parseable by CLI
+    """
+    application = 1
+    volume = 2
+    network = 3
+    secret = 4
+    secretValue = 5
+    gateway = 6
+
+def get_resource_type(file_name):

```

Comment:
Actually, it looks like this function is in sfmergeutility? Same with a lot of these functions. Can we just import the function from there and use it here? If so, please port over the requested changes here and let's remove the duplicate code :) Thanks!

Reply:
moved the code accordingly